networkx/generators/classic.py

```diff
@@ -103,6 +104,52 @@ def full_rary_tree(r, n, create_using=None):
     return G
 
 
+@nx._dispatchable(graphs=None)
+def kneser_graph(n, k):
+    """Returns the Kneser Graph with parameters `n` and `k`.
+
+    The Kneser Graph has nodes that are k-tuples (subsets) of the integers
+    between 0 and ``n-1``. Nodes are adjacent if their corresponding sets are disjoint.
+
+    Parameters
+    ----------
+    n: int
+        Total number of elements.
```

Comment:
yes I agree with you, however, I am not too sure what an alternative could be. 
Perhaps we could change it to ``Size of the universal set``? or simply ``Size of the set``. Since we do describe what the set and subset represent in the description above.
Let me know what you think.

Reply:
Number of integers from which to make subsets representing nodes.

Reply:
> Number of integers from which to make subsets representing nodes.

I think this is good, especially if it can be accompanied by an additional code explanation. Something like:

```
Number of integers from which to make node subsets.
Subsets are drawn from ``set(range(n))``.
```

(Assuming this is correct).

Reply:
this description does look better to me. I'll make the changes now